lib/ui/shared/bottom_sheets/profile_botton_sheet/custom_user_bottom_sheet_viewmodel.dart

```diff
@@ -1,18 +1,32 @@
+import 'package:hng/services/current_user_profile.dart';
+import 'package:hng/services/local_storage_services.dart';
+import 'package:hng/utilities/storage_keys.dart';
 import 'package:stacked/stacked.dart';
 import 'package:stacked_services/stacked_services.dart';
 
 import '../../../../app/app.locator.dart';
 import '../../../../app/app.router.dart';
 import '../../../../models/user_model.dart';
-import '../../../../package/base/server-request/api/http_api.dart';
-import '../../../../services/local_storage_services.dart';
-import '../../../../utilities/constants.dart';
-import '../../../../utilities/storage_keys.dart';
 
-class CustomUserBottomSheetViewModel extends FutureViewModel {
+final storageService = locator<SharedPreferenceLocalStorage>();
+
+class CustomUserBottomSheetViewModel extends BaseViewModel {
+  String photoUrl =
+      storageService.getString(StorageKeys.currentOrgId).toString();
+
+  String name = storageService.getString(StorageKeys.firstName).toString();
+
+  String displayName =
+      storageService.getString(StorageKeys.displayName).toString();
+
+  String phoneNum = storageService.getString(StorageKeys.phone_num).toString();
```

Comment:
The way you are passing data to get to this view is not proper, you are making too many storage request. Why not when the view loads, it get it from the api and show it

Reply:
> The way you are passing data to get to this view is not proper, you are making too many storage request. Why not when the view loads, it get it from the api and show it

The data is passed to the storage after login to decrease load time. The backend takes long to respond so the data is stored immediately after login